internal/dev_server/ui/asset_handler.go

```diff
@@ -0,0 +1,11 @@
+package ui
+
+import (
+	"embed"
+	"net/http"
+)
+
+//go:embed index.html
```

Comment:
@kwatkins-ld you can add more assets to the embed if you list them here. Take a look at these [docs](https://pkg.go.dev/embed#hdr-Directives) for the syntax.